Cargo.lock

```diff
@@ -2987,9 +2987,9 @@ checksum = "33a33a362ce288760ec6a508b94caaec573ae7d3bbbd91b87aa0bad4456839db"
 
 [[package]]
 name = "libmdbx"
-version = "0.1.4"
+version = "0.1.5"
```

Comment:
We need to pin MDBX at v0.1.4 for now to maintain Windows and macOS compatibility, which has been removed upstream (https://github.com/vorot93/libmdbx-rs/commits/master). This is a temporary solution while we work out whether we want to maintain a fork of MDBX or switch to another database (that's a bigger discussion).